src/projectile.cpp

```diff
@@ -167,3 +171,41 @@ int max_aoe_size( const std::set<std::string> &tags )
     }
     return aoe_size;
 }
+
+void multi_projectile_hit_message( Creature *critter, int hit_count, int damage_taken,
+                                   std::string projectile_name, std::string projectile_name_plural )
```

Comment:
I think it'd better to let caller handle plural translations and pass translated string as a single parameter here, like
```cpp
// caller
multi_projectile_hit_message( ..., n_gettext( "fragment", "fragments", n ), ... );
```

Otherwise gettext won't recognise fragment and fragments is a pair of singular and plural form of the same projectile name.

Reply:
Yes that sounds better, thanks.